data/pilots/galactic-empire/tie-in-interceptor.json

```
@@ -450,6 +450,26 @@
       "standard": true,
       "extended": true,
       "epic": true
+    },
+    {
+      "name": "Maus Monare",
+      "caption": "Battle Over Endor",
+      "xws": "mausmonare-battleoverendor",
+      "initiative": 3,
+      "limited": 1,
+      "shipAbility": {
+        "name": "Sensitive Controls",
+        "text": "During the System Phase, you may perform a red [Barrel Roll] or red [Boost] action."
+      },
+      "cost": 5,
+      "standardLoadout": ["noescape-battleoverendor", "outmaneuver-battleoverendor", "fuelinjectionoverride-battleoverendor"],
```

`Outmaneuver` here is identical to the customizable talent, so we can just use that XWS ID.

```suggestion
      "standardLoadout": ["noescape-battleoverendor", "outmaneuver", "fuelinjectionoverride-battleoverendor"],
```